portfolio/src/main/webapp/style.css

```diff
@@ -6,7 +6,7 @@ html {
 
 .jumbotron {
     align-items: center;
-    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumbo4.jpg");
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7),rgba(0, 0, 0, 0.7)),url("/images/jumbo4.jpg");
```

Comment:
Sorry to be nit-picky but don't forget to put spaces after the other commas as well

Reply:
No problem, hopefully I got all of the style changes this time.